### ISSUE TITLE
Technical improvements

### ISSUE THREAD

**Issue by samanthaburton** [samanthaburton/whose_data#24]
_Thanks so much for your interest in contributing to Whose Data?
If you haven't read the [Contributor Guidelines](https://github.com/samanthaburton/whose_data/blob/master/CONTRIBUTING.md) yet, please have a look before diving in._ 

I'm no expert in web design, data visualization or data management -- and the project ([www.whosedata.ca](whosedata.ca)) sure could use some help in these areas.

Here are some ideas I have for improvements, I'd welcome any support making them happen -- and/or to hear other ideas you may have. Thank you!

## I'd love help with...
- **A more intuitive user interface**. I'm just using a basic wordpress template right now, and the parallax layout probably isn't the best for this type of content.
- **An improved way to display the data**. it's in a simple (and ugly) table right now, it would be great to make it more visually appealing and concise. Dreaming big, I think it'd also be cool to have users select what companies/organizations they're interested in when they land on the site so they can see those first -- but that's not so much a priority as it is a moonshot idea!
- **Suggestions about how to manage large amounts of qualitative data, for an open source project**. Right now, every 'case' is a separate Github issue and the table on the website needs to be updated manually. This is okay at the moment, with such a small amount of data, but I could foresee in the future it being helpful to have it all in a machine-readable format (eg a spreadsheet) that multiple people could contribute to (while still maintaining version control). And it would be helpful if we could link that to the data visualization, to make updating it easier... but I have no idea how to do these things!
- **Search engine optimization** How can we make the project easier to find?

**Comment by sogen** [comment]
Will work on logo first to get a sense of the identity. 
Can you name some logos that you like, and any ideas?

**Comment by samanthaburton** [comment]
Thanks so much @sogen for your offer to support with design! Hugely appreciated!!

I think quite a simple design would work best for this project, potentially drawing on the focus on data but without taking it too literally (eg. I don't think it'd make sense to have a computer be part of the logo).

Here are a few examples that I like: [1](https://99designs-blog.imgix.net/wp-content/uploads/2018/01/attachment_69358233-e1514860631942.jpeg?auto=format&q=60&fit=max&w=930), [2](https://99designs-blog.imgix.net/wp-content/uploads/2018/01/attachment_75079507-e1514861638178.png?auto=format&q=60&fit=max&w=930), [3](https://99designs-blog.imgix.net/wp-content/uploads/2018/01/attachment_60579104-e1514862424249.jpeg?auto=format&q=60&fit=max&w=930), [4](https://99designs-blog.imgix.net/wp-content/uploads/2018/01/attachment_89720653-e1514862452525.jpeg?auto=format&q=60&fit=max&w=930)

Let me know what you think, and if you have any questions. Thanks again!

**Comment by sogen** [comment]
nice logo selection, will work on it :)

**Comment by sogen** [comment]
opened ticket #41 for conversation about the logo

**Comment by DigitalDesignerOnline** [comment]
Hi There, I would be happy to assist, I can help with the WordPress Theme/Layout as ell as any custom page templates or custom post types etc you desire. I can also assist with the table display and layouts. We could make a separate issue/PR for each or compile them into one however you prefer to handle it. Let me know nad I can get started today. :)

**Comment by samanthaburton** [comment]
Hi @DigitalDesignerOnline! Great to meet you, and thanks so much for your offer to help :) I'm happy to work on whichever system is best for you (re separate issues VS compiling them into one) and am  excited to hear your recommendations about table displays and custom posts... I'm open to any and all suggestions you might have to make the site better (right now, it's just running the basic Wordpress twenty seventeen theme) - so please let me know what you need from me to get started! And thanks so much again!

**Comment by DigitalDesignerOnline** [comment]
I will be sending you some PR's tonight for you to review. I will also outline the various areas where we can surely work together to make an amazing online presence. Sorry for the delay in response, been celebrating the Canadian Thanksgiving for the past few days. I think I am stuffed more then the Turkey was lol

**Comment by DigitalDesignerOnline** [comment]
Better Web Design - I can develop you a custom WordPress Theme

An improved way to display the data - We can easily make your table responsive or for that matter display the data in another method such as a foreach loop and creating the layout in div tags as opposed to table tags. 

Suggestions about how to manage large amounts of qualitative data, for an open source project - Have you considered using a custom database table to hold your information for this purpose. You can then update/insert/delete table rows from the database through either your Dashboard or a frontend page if you prefer. 

I can see three distinct different issues we could separate these into 
1 Theme Design
2 Modify Populated Data for a more responsive design
3 Modify data tables to populate from wp database as opposed to manual insertion.

I can assist with all 3 areas and have your PR's ready for the morning for you to peruse. I can do the initial build on my server so you can see it live in action and then add the files to the PR I create for you.

Let me know if that works for you?

**Comment by DigitalDesignerOnline** [comment]
btw, It will be nice working with a fellow Torontonian ;)

**Comment by samanthaburton** [comment]
Happy Thanksgiving - I'm similarly full of turkey and pie! Lovely to be working with a fellow Canadian & Torontonian!

Thank you so much for your support, I love all of these suggestions :D I feel like 2 & 3 are the most pressing, but that's probably my utilitarian side speaking! A responsive table (or another method of display) would be excellent (I know it looks awful on mobile right now) and I really like the option of a custom database - especially as the amount of info grows. But of course, I'm not one to turn down a custom theme! I don't think we need anything fancy, though something more fit-for-purpose is always nice. 

Building it on your server first to be able to see it live sounds great. I'm also a bit new to using PRs for technical work (I've been using GitHub mostly for project management to date) so I might need a little guidance on that front. Will let you know if/when I have questions!

So in short - all this works and sounds fantastic, many, many thanks!
